### PR TITLE
Added UNIQUE constraint to the connection ID of contacts

### LINKED ISSUE #99: Add unique index to connections model
The [conenctions model](https://github.com/hyperledger/aries-endorser-service/blob/60ee01168045f0982f9d91eaf72ecc1c2a088da2/endorser/api/endpoints/models/connections.py) does not enforce a primary key on the connection id. This causes multiple conenctions with the same id to be - potentially - stored in the database, breaking functionality.

Example: ACA-Py sends a webhook for a new connection, and receives a 500 error as a response. It will try again to deliver the webhook for several times during which the endorser may store new records with the same id.

### PR DESCRIPTION
**Pull request by Gavinok**
This PR resolves #99

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -250,13 +250,13 @@ The Author agent (which is configured as multi-tenant) exposes its Admin API on
 Open a second bash shell (cd to the directory where you have checked out this repository) and run the BDD tests with:
 
 ```bash
-LEDGER_URL=http://localhost:9000 TAILS_SERVER_URL=http://localhost:6543 ./manage run-bdd
+LEDGER_URL=http://localhost:9000 TAILS_SERVER_URL=http://localhost:6543 poetry run ./manage run-bdd
```
Comment (esune) [inline README.md:253]: Why is `poetry run` needed here? The `manage` script should be able to be executed without this dependency.

**Review by esune — COMMENTED** [review]
Have you tested the scenario where this revision is applied to a service that - potentially - already has duplicate connection record keys? I am not sure whether we are in that situation already, but the migration (and the deployment) would fail in that case. At minimum, we need a bit of documentation explaining what to do if this happens.

File: README.md (hunk 1 above)
Comment (Gavinok) [reply README.md:253]: the `behave` executable is missing. We could move the `poetry run` into the manage script it's self.

**Review by esune — APPROVED** [review]

**Comment by esune** [comment]
@Gavinok changes look good, still need to validate migration in the scenario where there are duplicate connection IDs in the database before merging

File: README.md [hunk 2]
```diff
@@ -95,6 +95,20 @@ For example, to startup the Endorser to run exclusively within a docker network
 ENDORSER_ENV=testing ACAPY_ENDPOINT=http://host.docker.internal:8050 ./manage start-bdd --logs
 ```
 
+## BREAKING CHANGES
+
+Note that as of the latest version all connection_id's in the contact table is expected to be unique. If upgrading fails with an error similar to 
+```
+sqlalchemy.exc.IntegrityError: (psycopg2.errors.UniqueViolation) could not create unique index "contact_connection_id_key"
+```
+then you will need to manually delete these duplicate entries. To list them you can preform the following sql query
```
Comment (esune) [inline README.md:104]: Nit.: typo `preform`->`perform`

**Pull request merged**